base-api-client/src/main/java/com/cloudesire/platform/apiclient/response/error/ErrorResponse.java

```diff
@@ -39,9 +39,26 @@ public void addErrorHolder( ErrorResponseEntry errorHolder )
         errorHolders.add( errorHolder );
     }
 
+    /**
+     * @deprecated better use {@link #singleErrorHolder()} to inadvertently losing multiple error messages
+     */
+    @Deprecated
     public ErrorResponseEntry firstErrorHolder()
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)